pkg/resource/acl/hooks.go

```diff
@@ -14,9 +14,15 @@
 package acl
 
 import (
+	"context"
 	"github.com/pkg/errors"
 
 	"github.com/aws-controllers-k8s/runtime/pkg/requeue"
+	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
+	ackutil "github.com/aws-controllers-k8s/runtime/pkg/util"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
```

Comment:
the `svcsdk "github.com/aws/aws-sdk-go/service/memorydb"` should be in the second block since it's an external package